lib/dropbox_api/endpoints/content_upload.rb

```diff
@@ -12,7 +12,8 @@ def build_request(params, content)
       body = content
       headers = {
         'Dropbox-API-Arg' => JSON.dump(params),
-        'Content-Type' => 'application/octet-stream'
+        'Content-Type' => 'application/octet-stream',
+        'Content-Length' => content.size.to_s,
```

Comment:
What can be content here? File? String? IO? All of them?

Maybe it will make sense to use similar check to [old one](https://github.com/dropbox/dropbox-sdk-ruby/blob/1b33a9e0a91b9afc0e00ecf8c25b9f760ff2f2f5/lib/dropbox_sdk.rb#L195-L205)?